catalog/video/video-mpegts-stream/src/test/java/org/codice/alliance/video/stream/mpegts/TestUdpStreamMonitor.java

```diff
@@ -148,15 +151,21 @@ public void testSetTitle() {
                 .get(), is(title));
     }
 
+    @Test(expected = NoSuchElementException.class)
+    public void testGetNullStreamUri() {
+        assertThat(udpStreamMonitor.getStreamUri()
```

Comment:
This test might be clearer if you tested `assertThat(udpStreamMonitor.getStreamUri().isPresent(), is(false))`